### PR TITLE
Asyncio hub support for Python 3.7 to 3.9

### PR DESCRIPTION
**Pull request by itamarst**

### CODE REVIEW

File: eventlet/hubs/asyncio.py [hunk 1]
```diff
@@ -24,11 +24,11 @@ class Hub(hub.BaseHub):
 
     def __init__(self):
         super().__init__()
-        self.sleep_event = asyncio.Event()
         # The presumption is that eventlet is driving the event loop, so we
         # want a new one we control.
         self.loop = asyncio.new_event_loop()
         asyncio.set_event_loop(self.loop)
+        self.sleep_event = asyncio.Event()
```
Comment (4383) [inline eventlet/hubs/asyncio.py:31]: Why we need to move this line?

**Comment by 4383** [comment]
All the new jobs looks successful

File: eventlet/hubs/asyncio.py (hunk 1 above)
Comment (itamarst) [reply eventlet/hubs/asyncio.py:31]: In older Python versions the Event() would've been created with the wrong loop if it ran before `set_event_loop()`, and then it complained about mismatching loops.

Comment (4383) [reply eventlet/hubs/asyncio.py:31]: ack

**Review by 4383 — APPROVED** [review]
LGTM

**Pull request merged**